server/tests-py/remote_relationship_tests/Readme.md

```diff
@@ -0,0 +1,45 @@
+
+
+### Setup ###
+
+The test setup includes two Postgres databases with [sportsdb](https://www.thesportsdb.com/) schema and data, and two GraphQL engines running on the Postgres databases. Then one of the GraphQL engines is added as a remote schema to another GraphQL engine.
+
+The data will be same in both the databases. But the tables will reside in different database schema in-order to avoid GraphQL schema conflicts.
+
+The Python script `test_with_sportsdb.py` will help in setting up the databases, starting the Hasura GraphQL engines, and setting up relationships (both local and remote). This script will run databases on docker, and the GraphQL engines are run with `stack exec`.
+
+#### Setup GraphQL Engines ####
+
```

Comment:
Modified readme to add the instructions for installing dependencies in a virtual environment

I did not face the bdist_wheel error. Probably because I am using Python 3.7